src/modules/main/partials/editViewPartials/filter/FilterEntry.vue

```diff
@@ -129,10 +129,19 @@ export default {
 		},
 		magicFields() {
 			if (this.selectedColumn && this.selectedColumn.type.substr(0, 9) !== 'selection') {
+				const mf = []
```

Comment:
Nitpick: Would prefer a more clear variable name

```suggestion
				const fields = []
```